### DISCUSSION TITLE
I can't install uhydro32 in Windows 10

### DISCUSSION THREAD

**Discussion by LaloCotanda** [community/community#126158] [Programming Help]
### Select Topic Area

Question

### Body

Hello.

I need help because I can't install Uhydro in my laptop with Windows 10. I donwloaded the software from here: https://www.geologie.sachsen.de/uhydro-und-zusatzprogramme-26957.html but I couldn't. I'm trying with an old version but the same, not success. 

What can I do?

Thanks in advance.

**Comment by AsmitaMishra24** [comment]
Hey, @LaloCotanda!
1. Check System Requirements:
- Ensure your laptop meets the system requirements specified by Uhydro.
- Verify the operating system version, available memory, and other prerequisites.

2. Download from a Trusted Source:
- Make sure you’re downloading Uhydro from the official website or a trusted source.
- Avoid downloading software from suspicious websites.

3. Run as Administrator:
- Right-click on the Uhydro installer (usually an .exe file).
- Choose “Run as administrator” to elevate permissions.

4. Compatibility Mode:
- Right-click on the installer.
- Go to “Properties” > “Compatibility” tab.
- Check “Run this program in compatibility mode for” and choose an older Windows version (e.g., Windows 7 or 8).

5. Check for Antivirus or Firewall Interference:
- Temporarily disable antivirus or firewall during installation.
- Security software can sometimes block installation processes.

6. Event Viewer for Errors:
- Press Win + X and select “Event Viewer.”
- Look for errors related to Uhydro installation under “Windows Logs” > “Application.”

I hope this will be helpful.

**Reply by LaloCotanda** [reply]
Thank you for your help. I'll try what you said.

Best regards.

**Reply by AsmitaMishra24** [reply]
If the above answer was helpful, please mark it as the answer @LaloCotanda

**Comment by LiteBrite82** [comment]
Hi @LaloCotanda ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this software, such as the manufactures website. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
